install/kubernetes/cilium/README.md

```diff
@@ -24,14 +24,11 @@ Helm chart for Cilium
 | bandwidthManager | bool | `false` |  |
 | bpf.clockProbe | bool | `false` |  |
 | bpf.lbMapMax | int | `65536` |  |
-| bpf.mapDynamicSizeRatio | float | `0.0025` |  |
-| bpf.masquerade | bool | `true` |  |
```

Comment:
Note that we have a problem here: We can't specify the new options in the values.yaml file, because otherwise those values will override the defaults, which means we will ignore upgrade compatibility rules. We have no way at this point to tell the difference between the user explicitly configuring the option vs. the default being picked up from the values.yaml file. So then when we interpret the setting, we can't tell if this is an upgrade or a new cluster.

Hence we remove the option from the default `values.yaml` to avoid forcing new features onto existing users. The side effect is that we cannot generate the full set of  available options in this README file.

Reply:
A downside of this is that it would be easy to consider the list on `README.md` as the authoritative (and complete) list of options.

We don't have to hold up this PR, but maybe per-version `values.yaml` would be one way to address this?